components/alert/alert-toast.js

```diff
@@ -88,29 +93,42 @@ class AlertToast extends LitElement {
 
 			.d2l-alert-toast-container[data-state="opening"],
 			.d2l-alert-toast-container[data-state="closing"] {
-				transition-duration: 250ms;
+				transition-duration: 600ms;
 				transition-property: transform, opacity;
-				transition-timing-function: ease-in;
+				transition-timing-function: ease;
+			}
+
+			.d2l-alert-toast-container.d2l-alert-toast-container-not-bottom[data-state="closing"] {
+				transform: unset;
+				transition-property: opacity;
```

Comment:
Just so I understand: this is preventing toasts that aren't at the bottom from transitioning vertically away as they close?

Reply:
Correct, we just want them transitioning with opacity

Reply:
Cool. Not sure if it would be cleaner then to invert everything such that the CSS class is actually just applied to the bottom one and that's where the transform/transition-property is defined? Then you don't have to apply it to all of them and then remove it from the not-bottom ones. 🤷